google-cloud-storage/src/main/resources/reference.conf

```diff
@@ -20,3 +20,8 @@ alpakka.google {
     token-scope = deprecated // "https://www.googleapis.com/auth/devstorage.read_write" // default
   }
 }
+
+alpakka.gcs {
```

Comment:
I used `alpakka.gcs` here. `alpakka.google-cloud-storage` can also work and it doesn't conflict with `alpakka.google.cloud.storage`. It is also possible to do `alpakka.google.cloud.storage` however it overlaps with the previously deprecated settings, due to the way typesafe config works this shouldn't cause any problems it just might be confusing for users.